src/player.cpp

```diff
@@ -2955,7 +2955,7 @@ int player::read_speed( bool return_stat_effect ) const
     // Stat window shows stat effects on based on current stat
     const int intel = get_int();
     /** @EFFECT_INT increases reading speed */
-    int ret = 1000 - 50 * ( intel - 8 );
+    int ret = to_moves<int>( 1_minutes ) - to_moves<int>( 30_seconds ) * ( intel - 8 );
```

Comment:
I think this is wrong.  There should be a factor of 20 difference between them like there was before, also, based on your change farther down, 100 is actually 1 second.  I think that one is right (100 moves per turn, one turn per second), so the other two should be .5 seconds, and 10 seconds. Unless you meant to change them, but the PR description didn't imply you were.

Reply:
I did mean to change them. The numbers needed updating to account for the new turn times without the *6 constant further down. The only one wrong here is the 30_seconds which should be 3_seconds to preserve that factor of 20.